packages/components/src/color-palette/style.scss

```diff
@@ -1,7 +1,7 @@
 .components-color-palette__custom-color {
+	position: relative;
```

Comment:
This relative position was probably needed for the abs positioned pseudo element but isn't needed anymore.